lib/appium_capybara/driver/appium/driver.rb

```diff
@@ -78,6 +78,11 @@ def scroll_down
       browser.execute_script('mobile: scroll', direction: 'down')
     end
 
+    # new
+    def swipe(start_x, start_y, offset_x=0, offset_y=0, duration=200)
+      Appium::TouchAction.new(browser).swipe(start_x: start_x, start_y: start_y, offset_x: offset_x, offset_y: offset_y, duration: duration).perform
```

Comment:
thanks! but does this work?

Current touch action works by `swipe(start_x: 75, start_y: 500, end_x: 75, end_y: 20, duration: 500)`
`offset` is old style. (removed in ruby_lib 9.12 https://github.com/appium/ruby_lib/blob/master/CHANGELOG.md#v9120 . This library requires over v9.15)

And keyword arguments are prefered since users do not need to consider the order.
So, like:

```
def swipe(start_x:, start_y:, end_x: 0, end_y: 0, duration: 200)
    Appium::TouchAction.new(browser).swipe(start_x: start_x, start_y: start_y, end_x: end_x, end_y: end_y, duration: duration).perform
end
```

Reply:
@KazuCocoa Yes, it does work for me; and yes, I noticed it and updated in this pull request. Thank you 😊